qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestData.java

```diff
@@ -684,4 +678,60 @@ public String toString() {
 		public String datetime;
 		public String uuid;
 	}
+
+	public static class PatientGenerator {
+
+		private static final String[] PATIENT_GIVEN_NAMES = {"Alexandre", "Achint",
+				"Brittany", "Burke", "Cesar", "Cosmin", "Daniel", "Darius",
+				"David", "Ellen", "Émerson", "Evan", "Fernando", "Gabou",
+				"Glauber", "Hamish", "Louise", "Mário", "Mark", "Natália", "Neil",
+				"Neissi", "Nelice", "Rafal", "Renee", "Wyclif"};
+		private static final String[] PATIENT_FAMILY_NAMES = {"Barbosa", "Sethi",
+				"Eddy", "Mamlin", "Vortmann", "Ioan", "Kayiwa", "Jazayeri",
+				"Walton", "Ball", "Hernandez", "Waters", "Freire", "Mandy",
+				"Ramos", "Fraser", "Sécordel", "Areias", "Goodrich", "Arsand",
+				"Craven", "Lima", "Heck", "Korytkowski", "Orser", "Luyima"};
+		private static final String[] PATIENT_BIRTH_MONTH = MONTHS;
+		private static final String[] PATIENT_BIRTH_DAY = DAYS;
+		private static final String[] PATIENT_BIRTH_YEAR = YEARS;
+		private static final String[] PATIENT_GENDER = {"Male", "Female"};
+
+		public static org.openmrs.contrib.qaframework.helper.TestPatient generateTestPatient() {
+			org.openmrs.contrib.qaframework.helper.TestPatient p = new org.openmrs.contrib.qaframework.helper.TestPatient();
```

Comment:
Ohh how come i didnt recognise this. Use the TestPatient  class 
```suggestion
			TestPatient  testPatient = new TestPatient();
			Then import the package 
```

Reply:
@kdaud @sherrif10 have tried this but indeed it can't be refactored to such.........u can try it locally @sherrif10 ...possibly share what u get

Reply:
do you really need to put the class package within the method declaration?
```suggestion
public static org.openmrs.contrib.qaframework.helper.TestPatient generateTestPatient() {
}
```

Reply:
Yes @kdaud it was necessary... try it locally and see ....I think @sherrif10 had tried it here 

> Since this class is not coming from any package. We could simply use it as if it comes from the very class. Have tested out this and all is fine. Another option would be refactoring the entire code which might bring in more code duplicates. cc @kdaud Let me know your suggestion on this. Thanks @Muta-Jonathan

Reply:
@Muta-Jonathan does the method fails when you get rid of `org.openmrs.contrib.qaframework.helper` and rather use the one suggested [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/281#discussion_r771928486)?

Reply:
@kdaud it doesn't work ...

Reply:
what about when you remove the static key word from the return type?

Reply:
@kdaud it fails other implemented classes

Reply:
@Muta-Jonathan you can rather rename one of the class if the refactoring is causing two files to have the same name within TestData class.

Reply:
have renamed the class ...To TestPatientData ...how about that @kdaud